public/redirect.html

```diff
@@ -6,8 +6,8 @@
     This page is loaded after the authorize page has done it's work. This
     simply updates local storage and the rest of torii takes over from there -->
 
-    <!-- load arcgis-rest-js -->
-    <script src="http://localhost:4200/assets/vendor.js"></script>
+    <!-- load latest arcgis-rest-js from CDN -->
+    <script src="https://unpkg.com/@esri/arcgis-rest-auth"></script>
```

Comment:
For now let's revlock this to 1.1.1, so use https://unpkg.com/@esri/arcgis-rest-auth@1.1.1 or even https://unpkg.com/@esri/arcgis-rest-auth@1.1.1/dist/umd/arcgis-rest-auth.umd.min.js

I'd say we should switch to using a local script from the public tree before the next time we update the rest-js version.

Other than that, this PR LGTM.

Reply:
> For now let's revlock this to 1.1.1

done and done.